DEVOPS.md

```diff
@@ -0,0 +1,20 @@
+# TC - DevOps Flow
+
+This app is currently on [Openshift GOLD](https://console.apps.gold.devops.gov.bc.ca/).
+
+The flow is spread across four namespaces, one for each environment (Dev, Test, Prod), and Tools for builds. As well, the Postgres DB sites in an instance of [Patroni](https://github.com/bcgov/patroni-postgres-container) with [backups](https://github.com/BCDevOps/backup-container) (TODO).
+
+### Deployment
+For deployment in a fresh namespace, there are a number of make commands that should assist in the initial setup of the environment.
+
+First create the Patroni instance. This can be done by running `db-prep` to create the necessary secrets, then `db-create` to create the StatefulSet that pulls from the BCGov image of Patroni.
+
+Note that the latest tag provided from BCGov Patroni uses Postgres 12.4.
```

Comment:
It probably doesn't matter, but I wonder if we should downgrade to postgres 12.4 for our local db ?

Reply:
I think we should as well!